internal/clients/team2/social_capital.go

```diff
@@ -49,10 +51,52 @@ func (a *AgentTwo) updateTrustworthiness(agentID uuid.UUID, actualAction, expect
 
 }
 
-// func (a *AgentTwo) updateInstitution(agentID uuid.UUID) float64 {
+// Get the direction to the voted lootbox
+func (a *AgentTwo) GetVotedLootboxForces(lootboxID uuid.UUID) utils.Forces {
+	lootbox := a.gameState.GetLootBoxes()[lootboxID]
+	lootboxPositionX, lootboxPositionY := lootbox.GetPosition().X, lootbox.GetPosition().Y
+	agentPositionX, agentPositionY := a.GetLocation().X, a.GetLocation().Y
+	deltaX := lootboxPositionX - agentPositionX
+	deltaY := lootboxPositionY - agentPositionY
+	angle := math.Atan2(deltaY, deltaX)
+	normalisedAngle := angle / math.Pi
+	turningDecision := utils.TurningDecision{
+		SteerBike:     true,
+		SteeringForce: normalisedAngle - a.gameState.GetMegaBikes()[a.GetBike()].GetOrientation(),
+	}
+	return utils.Forces{
+		Pedal:   utils.BikerMaxForce,
+		Brake:   0.0,
+		Turning: turningDecision,
+	}
+}
 
-// 	// return 0.5 // This is just a placeholder value
-// }
+// Called by Events to obtain Event Value for update Institution
+// Assume what they broadcast is the truth
+// TODO: Obtain actual action performed from messaging
+// 1. Rule Adhereance (Follow leader biker/ dictator)
+func (a *AgentTwo) RuleAdhereanceValue(agentID uuid.UUID, expectedAction, actualAction utils.Forces) float64 {
+
+	magnitude := actualAction.Turning.SteeringForce
+	actualVector := forcesToVectorConversion(actualAction)
+	expectedVector := forcesToVectorConversion(expectedAction)
+	similarity := cosineSimilarity(actualVector, expectedVector)
+
+	normalisedTrustworthiness := similarity * magnitude
+	forceApplied := a.forces.Pedal
+	return normalisedTrustworthiness * forceApplied
```

Comment:
Only thing I'm not sure about is using the normalisedTrustworthiness/normalisedReputation to affect the new value of our agent's institution. 

Apart from that, it LGTM

Reply:
Actually logic is messed up as well lol.

Reply:
Fixed, good catch!